python/tvm/topi/x86/depthwise_conv2d.py

```diff
@@ -122,13 +122,24 @@ def depthwise_conv2d_NCHWc(cfg, data, kernel, strides, padding, dilation,
 
     strides = strides if isinstance(strides, (tuple, list)) else (strides, strides)
     HSTR, WSTR = strides
-    pad_top, pad_left, pad_down, pad_right = get_pad_tuple(padding, (filter_height, filter_width))
+    #pad_top, pad_left, pad_down, pad_right = get_pad_tuple(padding, (filter_height, filter_width))
```

Comment:
If this is not useful anymore, please remove it rather than keeping it commented out here. Same for other commented lines below.